onnx/defs/tensor/defs.cc

```diff
@@ -1259,6 +1259,20 @@ When `reduction` is set to "mul", `output` is calculated as follows:
     for idx in np.ndindex(update_indices):
         output[indices[idx]] *= updates[idx]
 
+When `reduction` is set to "max", `output` is calculated as follows:
```

Comment:
nit: may be we can combine all these 4 descriptions into one, as:
```
    output = np.copy(data)
    update_indices = indices.shape[:-1]
    for idx in np.ndindex(update_indices):
        output[indices[idx]] = reduction-op (output[indices[idx]], updates[idx])
where the reduction-op is +/*/max/min as specified.
```

Reply:
Thats a great suggestion